### PR TITLE
Adapt denominate ad fix tests (#356)

### PR DESCRIPTION
**Pull request by arhtudormorar**
* [*] Add devDependecies react and better-sqlite3

* [*] Make Denominate independent from redux store

* Merge with development

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "@types/qs": "6.9.7",
     "@typescript-eslint/eslint-plugin": "5.0.0",
     "@typescript-eslint/parser": "5.14.0",
+    "better-sqlite3": "7.6.2",
```
Comment (StanislavSava) [inline package.json:39]: ugh

File: src/UI/Denominate/index.tsx [hunk 2]
```diff
@@ -91,8 +94,14 @@ const DenominateComponent = (props: DenominateType) => {
...
 };
 
+/**
+ * @param props.egldLabel  if not provided, will fallback on **EGLD**
+ */
 export const Denominate = (props: DenominateType) => {
-  const egldLabel = props.egldLabel || getEgldLabel();
+  const egldLabel =
+    props.egldLabel ||
+    fallbackNetworkConfigurations[EnvironmentsEnum.mainnet].egldLabel; // EGLD
```
Comment (StanislavSava) [inline src/UI/Denominate/index.tsx:103]: I wonder if we should extract this into a simple constant and use it inside configs, like we do with chainIds

File: src/UI/Denominate/tests/denominateComponent.test.tsx [hunk 3]
```diff
@@ -22,7 +22,8 @@ describe('Denominate component when decimals = 2', () => {
       value: '9999979999800000000000000',
       showLastNonZeroDecimal: false,
       showLabel: true,
-      decimals: 2
+      decimals: 2,
+      egldLabel: 'EGLD'
```
Comment (StanislavSava) [inline src/UI/Denominate/tests/denominateComponent.test.tsx:26]: We should probably import this from constants

File: src/UI/Denominate/index.tsx (hunk 2 above)
Comment (arhtudormorar) [reply src/UI/Denominate/index.tsx:103]: it's only this place we are using it like this. We can extract it to a constant if there will be one more place like this

File: package.json (hunk 1 above)
Comment (arhtudormorar) [reply package.json:39]: yes to fix tests for now

File: src/UI/Denominate/tests/denominateComponent.test.tsx (hunk 3 above)
Comment (arhtudormorar) [reply src/UI/Denominate/tests/denominateComponent.test.tsx:26]: it's OK since it's a test file

**Review by CiprianDraghici — APPROVED** [review]

**Review by StanislavSava — APPROVED** [review]

**Pull request merged**